### DISCUSSION TITLE
Fluent-plugin-elasticsearch 5.4.3

### DISCUSSION THREAD

**Discussion by yakovbeder** [fluent/fluentd#4548] [Q&A]
Good day, 
It seems that there many fixes applied in Fluent-plugin-elasticsearch 5.4.3, however the version used in the latest image is still 5.3.0.
Is it planned to update the plugin version ? Also , how can I build an image with the latest version of the plugin?
Thank you advance

**Comment by kenhys** [comment]
Do you mean https://hub.docker.com/r/fluent/fluentd-kubernetes-daemonset/?
If so, these images are maintained under https://github.com/fluent/fluentd-kubernetes-daemonset.
Feedback (Pull Request) is welcome.

**Reply by yakovbeder** [reply]
Thank you for your reply, I was referring to the GitHub repository. I'll try to make a PR.
